libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/internal/navigation/HttpClientTest.kt

```diff
@@ -41,22 +48,36 @@ class HttpClientTest {
 
     @Test
     fun `check success network response`() {
+        val latch = CountDownLatch(1)
+        every { httpResponseCallback.run(any(), any()) } answers {
+            latch.countDown()
+        }
         val mockResponse = buildResponse(SUCCESS_CODE, SUCCESS_BODY)
         mockServer.enqueue(mockResponse)
-        val response = httpClient.executeMockRequest()
+        httpClient.executeMockRequest(httpResponseCallback)
+
+        if (!latch.await(5, TimeUnit.SECONDS)) {
+            Assert.fail()
```

Comment:
NIT

```suggestion
            fail()
```

Same thing applies 👇